integration-tests/go.mod

```diff
@@ -4,12 +4,12 @@ go 1.18
 
 require (
 	github.com/ethereum/go-ethereum v1.10.16
-	github.com/onsi/ginkgo/v2 v2.1.3
+	github.com/onsi/ginkgo/v2 v2.1.4
 	github.com/onsi/gomega v1.19.0
 	github.com/rs/zerolog v1.26.1
 	github.com/satori/go.uuid v1.2.0
-	github.com/smartcontractkit/chainlink-testing-framework v1.1.22-0.20220511162451-b8f31235228e
-	github.com/smartcontractkit/helmenv v1.0.67
+	github.com/smartcontractkit/chainlink-testing-framework v1.2.0
```

Comment:
I ran `go get github.com/smartcontractkit/chainlink-testing-framework@v1.2.0` which updated all the dependencies